source/reference/minio-mc/mc-support-top-net.rst

```diff
@@ -0,0 +1,75 @@
+=======================
+``mc support top net``
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc support top net
+
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-minio-only
+   :end-before: end-minio-only
+
+Syntax
+------
+
+.. start-mc-support-top-net-desc
+
+The :mc:`mc support top net` command displays realtime network metrics.
+
+.. end-mc-support-top-net-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command displays the current realtime network metrics for the :term:`alias` ``myminio`` deployment.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc support top net myminio/
+
+      The output returns information such as the server URL, interface, receive, transmit, and messages.
```

Comment:
Nit - this reads a little weird?

```suggestion
      The output returns information such as the server URL, network interface, receive rate, transmit rate, and messages.
```

Which...what are messages? but eh maybe its fine.